tutorials/epochs/40_autogenerate_metadata.py

```diff
@@ -490,8 +526,6 @@
 ax.set_title("Channel: FCz")
 fig.suptitle("ERN (Difference Wave)", fontweight="bold")
 
-fig
```

Comment:
We should do here whatever we decide above

Reply:
I'd just remove the `fig` line.